### DISCUSSION TITLE
Feature Request: Require Confirmation to Trash or Archive

### DISCUSSION THREAD

**Discussion by rickgordon** [protz/thunderbird-conversations#2042] [Ideas]
I have, on a couple of occasions, accidentally clicked the Archive or Trash button in the title bar, and found that they operate without requiring confirmation. 

I would welcome an option to require confirmation before doing that, as I would most likely be doing intended trashing/archiving by other means, so some means of foiling an accidental click would be good.

**Comment by Standard8** [comment]
I think this is more Thunderbird's responsibility rather than Conversations. Indeed, there's already a request there for delete: https://bugzilla.mozilla.org/show_bug.cgi?id=895140. Not having this is slightly offset by having undo functionality.

Either way, I don't think this is Conversations' responsibility to provide.
